firmware/arduino/chessboard/chessboard.ino

```diff
@@ -98,6 +98,9 @@ void setup()
 
   stepsPerUnitSpace = MILLIMETERS_PER_UNITSPACE * STEPS_PER_MILLIMETER;
 
+  currentX = 0;
```

Comment:
Can you add a comment here explaining that you're initializing them to 0 here for safety?